lib/Model.js

```diff
@@ -77,7 +77,9 @@ class Model {
         let { via } = vulnerability;
 
         if (typeof via[0] === "string") {
-          via = parsedOutput.vulnerabilities[via[0]].via;
+          while (typeof via[0] === "string") {
+            via = parsedOutput.vulnerabilities[via[index] || via[0]].via;
+          }
```

Comment:
While it's fine as-is, this could technically be a `do...while` instead since you've already checked the condition above.

```js
if (typeof via[0] === "string") {
  do {
    via = parsedOutput.vulnerabilities[via[index] || via[0]].via;
  } while (typeof via[0] === "string");
}

Reply:
Fixed :)